docs/en/introduction.md

```diff
@@ -0,0 +1,99 @@
+# Introduction
+
+> In this chapter, we will use the Ribir syntax to write some simple examples. You only need to understand the general idea, no need to go into details. We will introduce it in detail in the following chapters.
+
+## What is Ribir?
+
+Ribir is a Rust framework for building cross-platform user interfaces. It allows you to compile applications for desktop, mobile and browser with one code. Ribir uses a non-intrusive declarative programming model that allows you to develop and design user interfaces as an independent module.
+
+Its core design concept is:
+
+> The UI is a re-description of data interaction and continues to respond to modifications of data.
+
+We emphasize "re" here because we think the API is the first description of data interaction, and building UI with Ribir requires only the data API.
+
+## Why choose Ribir?
+
+### Non-intrusive programming model
+
+Ribir only interacts with the API of your data and does not require any pre-design of your data for the user interface: no additional states, no additional notification mechanisms, no inheritance of any base classes or any other pre-constraints. It doesn't break the logic and structure of your existing data, nor does it inject any additional objects. When developing the core part of an application, you can focus on designing the data, logic, and API of the application without thinking about the UI at all.
```

Comment:
It doesn't break the logic and structure of your existing data, nor does it inject any additional objects. 
-> 
It doesn't break the logic and structure of your existing data or inject any additional objects.